src/query/settings/src/settings_default.rs

```diff
@@ -309,6 +309,12 @@ impl DefaultSettings {
                     mode: SettingMode::Both,
                     range: Some(SettingRange::Numeric(0..=u64::MAX)),
                 }),
+                ("spilling_use_parquet", DefaultSettingValue {
```

Comment:
let's rename it to `spilling_data_format` which ranges from [  "arrow, "parquet" ]